test/PR_test/integration_test/op/numpyop/meta/test_fuse.py

```diff
@@ -41,6 +47,14 @@ def instantiate_system():
 
         self.assertEqual(loaded_var, new_var)
 
+    def test_delete_op(self):
+        ops = fe.op.numpyop.meta.Fuse(
+            ops=[TestNumpyOp(inputs='x', outputs='x', mode="train", var=1), Delete(
+                keys='x',
+                mode="train",
+            )])
+        _ = ops.forward(data=[self.tf_data], state={})
```

Comment:
need to assert that key `x` is indeed deleted

Reply:
With the changes this test case is already failing but will add that assert too.

Reply:
the test case is failing?

Reply:
I mean without the changes the unit test would fail.